teuthology/suite/run.py

```diff
@@ -119,8 +119,9 @@ def choose_kernel(self):
         # Put together a stanza specifying the kernel hash
         if self.args.kernel_branch == 'distro':
             kernel_hash = 'distro'
-        # Skip the stanza if no -k given
-        elif self.args.kernel_branch is None:
+        # Skip the stanza if '-k none' is given
+        elif self.args.kernel_branch is None or \
+             self.args.kernel_branch.lower() == 'none':
```

Comment:
Nit: I don't think converting to lower case is necessary -- we don't do that for `distro`.

Reply:
Thanks for the comment. Sometimes people may wish to provide None as a in python, it is a special type constant, that's why I would prefer to match both 'None' and 'none'.